index.xml

```diff
@@ -422,17 +473,18 @@ Note that the use of the `Digest` header field is per
 target="http://tools.ietf.org/html/rfc3230#section-4.3.2">Section 4.3.2</eref>
 and is included merely as a demonstration of how an implementer could include
 information about the body of the message in the signature.
-The following sections also assume that the "rsa-key-1" keyId refers to a
-private key known to the client and a public key known to the server.
-The "hmac-key-1" keyId refers to key known to the client and server.
+The following sections also assume that the "rsa-key-1" keyId asserted by the
+client is an identifier meaningful to the server.
    </t>
 
    <section anchor="auth-isa" title="Initiating Signature Authorization">
 
     <t>
-A server may notify a client when a protected resource could be accessed by
-authenticating itself to the server. To initiate this process, the server
-will request that the client authenticate itself via a 401 response code. The
+A server may notify a client when a resource is protected by requiring a signature,
+To initiate this process, the server
```

Comment:
should that be a full stop above?

Reply:
Resolved 54dc50addae904b8a42cd152456bf53fdf438256